server/repository/src/db_diesel/stocktake_row.rs

```diff
@@ -81,6 +85,8 @@ impl Default for StocktakeRow {
             inventory_reduction_id: Default::default(),
             is_locked: Default::default(),
             program_id: Default::default(),
+            counted_by: Default::default(),
+            verified_by: Default::default(),
```

Comment:
huh, i thought we cleaned all these defaults up (like just deriving default on stocktake row, and annotating any necessary defaults for funky field types...) 🤷‍♀️

Reply:
It's because of the datetime defaults ;3 https://doc.rust-lang.org/nightly/core/default/trait.Default.html#tymethod.default

Reply:
I think naive date time implements default now? This compiles fine https://github.com/msupply-foundation/open-msupply/compare/6995-Stocktake-Add-a-Performed-by-field...6995-Stocktake-Add-a-Performed-by-field(defaults)

Reply:
Yeah, I tried that as well, but unsure how it was before, so... well yolo lol